src/styles/block/stopwatch.scss

```
@@ -0,0 +1,36 @@
+.stopwatch {
+  margin: auto;
+  display: flex;
+  position: relative;
+  @include size (700px, 700px);
+  @extend %stopwatch-border-style;
+
+  &__arrow-small {
+    position: absolute;
+    bottom: 50%;
+    left: 49%;
```

I believe that you can use transform translate and leave here 50%

```suggestion
    left: 50%;
```